frontend/src/Change/ChangeInfo.css

```diff
@@ -0,0 +1,127 @@
+html, body {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  overflow-x: hidden; /* Prevent horizontal scrolling */
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #ABD2FA;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.1);
+  padding: 0;
+  border-radius: 0;
+  height: 100%; /* Full height of the viewport */
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center; /* Center the content vertically */
+  width: 100%;
+  height: 100%; /* Full height of the viewport */
+  background: #ABD2FA;
+  padding: 20px;
+  border-radius: 0; /* No border radius */
+}
+
+.form-container {
+  display: flex;
+  gap: 50px; /* Space between the forms */
+}
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  width: 600px;
```

Comment:
instead of using pixel css, make sure you use % or enum because pixel css is dependent on screens.